src/internet_identity/src/main.rs

```diff
@@ -782,12 +781,12 @@ fn get_signature(
         ));
     }
 
-    let tree = hashtree::fork(
-        HashTree::Pruned(hashtree::labeled_hash(
+    let tree = ic_certified_map::fork(
+        HashTree::Pruned(ic_certified_map::labeled_hash(
             LABEL_ASSETS,
             &asset_hashes.root_hash(),
         )),
-        hashtree::labeled(&LABEL_SIG[..], witness),
+        labeled(&LABEL_SIG[..], witness),
```

Comment:
nit: is there a reason to import `labeled` directly instead of using `ic_certified_map::labeled` as in the other places?

Reply:
No, changed to ic_certified_map::labeled